src/wwwroot/services/settings.js

```diff
@@ -231,5 +232,25 @@
         url: RELAY_CONFIG.cuitServiceBaseUrl + '/taxinfo/by-cuit/' + cuit
       });
     }
+
+    function resubscribeEmailAddress (subscriberEmail, reason, domainName, onExpectedError) {
+      var url = RELAY_CONFIG.baseUrl
+        + '/accounts/' + auth.getAccountName()
+        + '/subscribers/'
+        + subscriberEmail
+        + '/resubscribe';
+
+      var data = {};
+      data['reason'] = reason;
+      data['domain'] = domainName;
+
+      return $http({
+        actionDescription: 'action_requesting_resubscribe_email_address',
+        //tryHandleError: function(rejection){ return tryHandleErrorDomainManager(rejection, onExpectedError); },
+        method: 'POST',
+        data: data,
```

Comment:
```suggestion
        data: {
        reason: reason,
        domain: domainName
      },
```
just to continue the convention of most other methods in the file